docs/DeveloperGuide.md

```diff
@@ -552,21 +552,99 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Command Testing
+#### Adding A Person or A Company
+1. Adding a person
 
-1. Deleting a person while all persons are being shown
+    1. Test case: `add p n/Tom p/12345678 e/tom@gmail.com a/Kent Ridge Road t/friend`<br>
+       Expected: A person contact with the given information is added to the end of the person contact list. 
+            Details of the newly added person contact shown in the status message. Timestamp in the status bar is updated.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    1. Test case: `add p n/Tom`<br>
+       Expected: No person is added due to field missing(Only the tag field with `t/` prefix is optional). Error details shown in the status message. Status bar remains the same.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    1. Other incorrect add person commands to try: `add p`, `add p n/Tom p/12345678`, `...` (where compulsory fields are missing)<br>
+       Expected: Similar to previous.
 
-   1. Test case: `delete 0`<br>
+2. Adding a company
+
+    1. Test case: `add c n/ByteDance p/12345678 e/bytedance@example.com d/Cool Company t/social media`<br>
+       Expected: A company contact with the given information is added to the end of the company contact list.
+       Details of the newly added company contact shown in the status message. Timestamp in the status bar is updated.
+
+    1. Test case: `add c n/ByteDance`<br>
+       Expected: No company is added due to field missing(Only the tag field with `t/` prefix is optional). Error details shown in the status message. Status bar remains the same.
```

Comment:
Missing space between 'missing' and '(Only...'. Will merge and correct it on my end in the interest of time

Reply:
Also fixed in the new commit.